NDTensors/src/dense/set_types.jl

```diff
@@ -31,8 +8,9 @@ function set_datatype(storagetype::Type{<:Dense}, datatype::Type{<:AbstractArray
   )
 end
 
-function set_eltype(
-  storagetype::Type{<:Dense{ElT,DataT}}, ElR::Type
-) where {ElT,DataT<:AbstractArray}
-  return Dense{ElR,similartype(DataT, ElR)}
-end
+## TODO move to UnspecifiedArrays
+# function set_eltype(
+#   storagetype::Type{<:Dense{ElT,DataT}}, ElR::Type
+# ) where {ElT,DataT<:AbstractArray}
+#   return Dense{ElR,similartype(DataT, ElR)}
+# end
```

Comment:
Reminder to delete this if it is no longer needed.

Reply:
Scratch that, I think we will need a specialized definition for `Dense` since it is a wrapper type, see my comment below about the `set_eltype` and `set_ndims` definitions for `AbstractArray`. Though it seems like this definition could be simplified by making use of `set_eltype`/`set_parameters`.